src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GoToGCPConsoleStep.ts

```diff
@@ -15,7 +15,7 @@ interface GoToGCPConsoleStepProps {
 
 export const GoToGCPConsoleStep = ({ isActive, ...props }: GoToGCPConsoleStepProps) => {
   return h(Step, { isActive }, [
-    StepHeader({ title: 'STEP 1' }),
+    h(StepHeader, { title: 'STEP 1' }),
```

Comment:
As a component, StepHeader should always be rendered using `h` instead of called as a function. If called as a function, there is no React Element created for StepHeader, so things like hooks would not work (if a hook was later added to StepHeader). [Render functions](https://reactjs.org/docs/render-props.html) do have their place, but by convention, their name should start with a lower case letter (and preferably start with "render").